packages/driver/src/cypress.ts

```diff
@@ -37,7 +37,7 @@ import ProxyLogging from './cypress/proxy-logging'
 import * as $Events from './cypress/events'
```

Comment:
Commenting here because I can't comment on file-renames, but shouldn't  the file contents of
`src/cy/origin` be under `src/cy/commands/origin` to keep all commands code scope to that dir? I know net-stubbing is standalone, but I think it should also be moved 😅

Reply:
It would also remove the need to [add this import ](https://github.com/cypress-io/cypress/pull/21231/files#diff-8ca2c866dca492abea845c9ef35963f7a539e0ad761cdca8b0ac0a1e408e27b0R4) in `driver/src/cypress/commands.ts`:

Reply:
addressed in [8f009be](https://github.com/cypress-io/cypress/pull/21231/commits/8f009be59275235241b19fe50ea33d86318ecc37) and [a74c6fb](https://github.com/cypress-io/cypress/pull/21231/commits/a74c6fbc4d77bcb7fd63d1aea682bf3e7c962961)